cobrawap/pipeline/stage04_wave_detection/Snakefile

```diff
@@ -121,30 +121,30 @@ use rule template as plot_waves with:
 
 use rule template as time_slice with:
     input:
-        data = Path('{dir}') / f'{{data_name}}.{config.NEO_FORMAT}',
+        data = Path('{dir}') / str('{data_name}' + "." + config.NEO_FORMAT),
         script = SCRIPTS / 'time_slice.py'
     params:
-        params()
+        params(t_start=config.SLICE_TSTART,t_stop=config.SLICE_TSTOP)
     output:
-        Path('{dir}') / f'{{data_name}}_{{t_start, [0-9]+}}-{{t_stop, [0-9]+}}s.{config.NEO_FORMAT}'
+        Path('{dir}') / str('{data_name}' + ("_%s-%ss." % (config.SLICE_TSTART, config.SLICE_TSTOP)) + config.NEO_FORMAT)
```

Comment:
same issue as in previous comment. The rule should maintain its flexibility by setting the t_start and t_stop values via the wildcards and not the config values

Reply:
Like this, you would need to still write the snakemake call with the t_start and t_stop times in the filename, but now it only works if it matches the config settings